packages/next/build/webpack-config.ts

```diff
@@ -432,8 +432,7 @@ export default async function getBaseWebpackConfig(
             // are relative to requests we've already resolved here.
             // Absolute requires (require('/foo')) are extremely uncommon, but
             // also have no need for customization as they're already resolved.
-            const start = request.charAt(0)
-            if (start === '.' || start === '/') {
+            if (/^[./]/.test(request)) {
```

Comment:
```suggestion
            if (request.startsWith('.') || request.startsWith('/')) {
```

What you had initially was much more readable/understandable when looking at the code.

Reply:
Reverted back to the initial commit.